controllers/vrg_kubeobjects.go

```diff
@@ -168,8 +168,8 @@ func (v *VRGInstance) kubeObjectsCaptureStartOrResumeOrDelay(
 }
 
 func kubeObjectsCaptureStartConditionallySecondary(
-	v *VRGInstance, result *ctrl.Result,
-	captureStartGeneration int64, captureStartTimeSince, captureStartInterval time.Duration,
+	v *VRGInstance, _ *ctrl.Result,
```

Comment:
Can remove these parameters.

Reply:
This function must have a signature:
captureStartConditionally     func(*VRGInstance, *ctrl.Result, int64, time.Duration, time.Duration, func())